tests/Validation/ValidatorTest.php

```diff
@@ -660,4 +660,100 @@ public function getOptionValue(string $name): string {
 			],
 		],  
 	],
+]);
+
+test('validate yields valid results for files', function ($expected, $params, $files, $formId, $formData) {
+	expect($this->validator->validate($params, $files, $formId, $formData))->toBe($expected);
+})->with([
+	[
+		[
+			'file-wrong-type' => 'The file type is not supported. Only .pdf are allowed.',
```

Comment:
Sounds weird a bit :D 
```suggestion
			'file-wrong-type' => 'The file type is not supported. Only .pdf is allowed.',
```

Or maybe "only .pdf's are allowed", "only .pdf files are allowed"

Reply:
I agree that there's work to be done regarding these labels, but as these are tests, I've went ahead and wrote expectations to be in line with currently implemented labels. I can work on these labels in future commits, but I'd like to focus on increasing test coverage.

I've also considered relying on the appropriate Labels class here (instead of string constants) - not sure what's the better approach here. (cc @dingo-d)

Reply:
Did we discuss this last week? My memory is shoddy 🙈